docs/build-process/worker-build-process.md

```diff
@@ -0,0 +1,90 @@
+# Building a worker executable
+This page explains how to build a [worker](https://docs.improbable.io/reference/latest/shared/glossary#worker) in a SpatialOS project using the flexible project layout (FPL). It also serves as a guide on how to migrate the build process of a worker from the [current project layout (CPL)](https://docs.improbable.io/reference/latest/shared/reference/project-structure) to the FPL. Refer to our [Migration guide](../migration-guide/migration-guide-master-page.md) for a summary of additional migration steps.
+
+We provide a reference implementation for building a C# worker that includes all worker build steps described in this guide. You can find it in the form of a shell script [here](../../build.sh).
+
+## Worker build process overview
+To build a worker in the FPL, you need to:
+1. [Download dependencies](#1-download-dependencies)
+2. [Generate code](#2-generate-code)
+3. [Build the worker executable](#3-build-the-worker-executable)
+
+**Migration advice:** With the CPL, the spatial CLI performs the worker build steps listed above through a combination of CLI commands ([`spatial worker codegen`](https://docs.improbable.io/reference/latest/shared/spatial-cli/spatial-worker-codegen), [`spatial worker build`](https://docs.improbable.io/reference/latest/shared/spatial-cli/spatial-worker-build)) and configuration files ([`worker build configuration`](https://docs.improbable.io/reference/latest/shared/worker-configuration/worker-build), [`spatialos_worker_packages.json`](https://docs.improbable.io/reference/latest/shared/reference/file-formats/spatial-worker-packages)). With the FPL, you need to perform all build steps manually which gives you full control over the build process.
```

Comment:
Do you think it's important to clarify that the worker build configuration isn't actually a file, but a section within a file? It might be clear enough as it is; I just noticed that it's not quite accurate.

Reply:
The CPL worker build config file is a fully stand-alone file.

Reply:
Ah, I thought you specified all the build-related stuff in the "build" field of the worker config file? At least that's what our docs seem to say: https://docs.improbable.io/reference/13.6/shared/worker-configuration/worker-build